package.json

```diff
@@ -64,6 +64,7 @@
   },
   "dependencies": {
     "classnames": "^2.1.2",
+    "history": "^1.17.0",
```

Comment:
I think it may not necessary for react-bootstrap-table, but example use it, maybe we can move it into devDependencies ?